rskj-core/src/main/java/co/rsk/rpc/netty/RskWebSocketServerProtocolHandler.java

```diff
@@ -15,7 +15,7 @@ public class RskWebSocketServerProtocolHandler extends WebSocketServerProtocolHa
 
     public RskWebSocketServerProtocolHandler(String websocketPath, int maxFrameSize) {
         // there are no subprotocols nor extensions
-        super(websocketPath, null, false, maxFrameSize, true);
+        super(websocketPath, null, false, maxFrameSize, true, true);
```

Comment:
@rmoreliovlabs one small thing, please add some comment explaining why we set this flag and the Netty limitation (`a port:host can only have one webSocketPath, and that's a Netty limitation.`)

This way the next dev changing this will be aware ;)

Reply:
Done